tools/system_libs.py

```diff
@@ -1137,7 +1137,7 @@ def get_files(self):
   def can_build(self):
     # ASMFS is looking for a maintainer
```

Comment:
Do you intend to work on ASMFS? If so let's remove this comment. Perhaps also enable tests?

Reply:
Yes, at some point. Not yet able to give it full attention, but this was enough to make my 3D engine work again on latest Emscripten master.

Checking out our asmfs tests in browser suite, unfortunately they do not yet work with this.

Reply:
I see, thanks. lgtm